src/common/exponent/exponentHelper.ts

```diff
@@ -124,12 +126,12 @@ var {Component} = React;
 var ReactNative = require('react-native');
 var {AppRegistry} = ReactNative;
 
-var entryPoint = require("../${entryPointFile}");
+var entryPoint = require("./${entryPointFile}");
 
 AppRegistry.registerRunnable('main', (appParameters) => {
     AppRegistry.runApplication('${componentName}', appParameters);
 });`;
-                return this.fileSystem.writeFile(this.dotvscodePath(EXPONENT_INDEX), fileContents);
+                return this.fileSystem.writeFile(this.pathToFileInWorkspace(EXPONENT_INDEX), fileContents);
```

Comment:
I'm not sure that we want to put this file in with the user's sources rather than in the `.vscode` folder; it's not really part of the end-user's app, it's just an implementation detail of us converting to an exponent app.